backend-models/service-tests/courseedservice-tests.txt

```diff
@@ -0,0 +1,258 @@
+- Insert into course (line 79)
+---------------------------------------------------------------------------------------------------------------------
+
+*********
+* pre-req:
+********* 
+
+{(checklogin) && ($ha)} IS TRUE
+
+*********
+* send parameters inputs to courseedservice.php 
+*********
+send
+{
+    $opt = "NEW";
+    $userid = "UNK"; or $userid = !NULL;
+    $coursecode = "G1337";
+    $coursename = "Testing-Course";
+    $courseGitURL = NULL; // There are no values in the database
+}
+
+*********
+* Gather service output:
+*********
+Save the values of the echoed array as a JSON (at the end of the file. echo json_encode($array);)
+this is the expected output for the micro service.
+
+---------------------------------------------------------------------------------------------------------------------
+- Insert into vers (line 116)
+---------------------------------------------------------------------------------------------------------------------
+
+*********
+* pre-req:
+********* 
+
+{(checklogin) && ($ha)} IS TRUE
+
+*********
+* send parameters inputs to courseedservice.php 
+*********
+send
+{
+    $opt = "NEWVRS";
+    $cid = 1885;
+    $coursecode = "G1337";
+    $versid = 1337;
+    $versname = NULL; // There are no value in the database
+    $coursename = "Testing-course";
+    $coursenamealt = "Course for testing codeviewer";
+    $motd = "Code examples shows both templateid and boxid!";
+    $startdate = "2020-05-01 00:00:00";
+    $enddate = "2020-06-30 00:00:00";
+}
+
+*********
+* Gather service output:
+*********
+Save the values of the echoed array as a JSON (at the end of the file. echo json_encode($array);)
+this is the expected output for the micro service.
+
+---------------------------------------------------------------------------------------------------------------------
+- Update course (line 138)
+---------------------------------------------------------------------------------------------------------------------
+
+*********
+* pre-req:
+********* 
+
+{(checklogin) && ($ha)} IS TRUE
+&& $makeactive = "3"
+
+*********
+* send parameters inputs to courseedservice.php 
+*********
+send
+{
+    $cid = 1885;
+    $versid = 1337;
+}
+
+*********
+* Gather service output:
+*********
+Save the values of the echoed array as a JSON (at the end of the file. echo json_encode($array);)
+this is the expected output for the micro service.
+
+---------------------------------------------------------------------------------------------------------------------
+- Update vers (line 152)
+---------------------------------------------------------------------------------------------------------------------
+
+*********
+* pre-req:
+********* 
+
+{(checklogin) && ($ha)} IS TRUE
+
+*********
+* send parameters inputs to courseedservice.php 
+*********
+send
+{
+    $opt = "UPDATEVRS";
+    $courseid = 1885;
+    $coursecode = "G1337";
+    $versid = 1337;
+    $versname = NULL; // There are no value in the database
```

Comment:
Cannot have the value NULL 
![image](https://user-images.githubusercontent.com/102600391/235904043-7d14cd1c-e321-43a6-b853-c8176dd500fb.png)
![image](https://user-images.githubusercontent.com/102600391/235904241-f5ec407d-7549-427b-afe9-9c27511a6214.png)